package.json

```diff
@@ -43,6 +43,7 @@
   "dependencies": {
     "bluebird": "^3.3.4",
     "classnames": "^2.1.5",
+    "keyboardevent-key-polyfill": "github:technologyadvice/keyboardevent-key-polyfill#0eb142e8d6845a90f23d21b468a3c8c8162b5a25",
```

Comment:
With every other event property you're used to using having been deprecated:

![image](https://cloud.githubusercontent.com/assets/5067638/14870529/bb69e878-0c8f-11e6-9bda-f6371d956a1c.png)

...and their successor property [`key`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) being only in working draft status with limited support:

![image](https://cloud.githubusercontent.com/assets/5067638/14870551/e251fa0c-0c8f-11e6-8ae6-cfec3beb8d51.png)

We are using a polyfill to ensure the `key` property is available on all keyboard events.  That said, this polyfill was missing the space key so we forked and updated.  Just [waiting on a release](https://github.com/cvan/keyboardevent-key-polyfill/issues/10) so we can go back to the official module.